### PR TITLE
feat: alby-tools browser script

### PR DESCRIPTION
**Pull request by rolznz**
Builds a new browser-compatible js file (index.browser.js) and exposes alby-tools exports on `window["alby-tools"]`

Demo: https://codepen.io/rolznz/pen/ZEmXGLd

### CODE REVIEW

**Review by bumi — COMMENTED** [review]
wahoo

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ or
 ```
 yarn add alby-tools
 ```
+or for use without any build tools:
+```
+// alby-tools now available at window["alby-tools"]
+<script src="https://cdn.jsdelivr.net/npm/alby-tools@3.0.0/dist/index.browser.js"></script>
```
[the remark this answers is not in the document]
Comment (rolznz) [reply README.md:21]: This would be available in the zip files generated by the release, right?

**Comment by rolznz** [comment]
@bumi @im-adithya are you guys ok for the functions to be accessed through `window["alby-tools"]`? I also added `window.albyTools` as an alternative

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bumi) [reply README.md:21]: no maybe just in the `dist` folder?

**Comment by bumi** [comment]
I'd only use `window.albyTools` unlikely that we use window["alby-tools"]. and we don't need both.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rolznz) [reply README.md:21]: Sorry, I don't understand. So the user builds it themselves? or where do we host it?

[the remark this answers is not in the document]
Comment (bumi) [reply README.md:21]: we just add it to the repo to download it. like in the good old days :)

**Pull request merged**